packages/cron/src/jobs/metrics.js

```diff
@@ -0,0 +1,119 @@
+import debug from 'debug'
+import settle from 'p-settle'
+
+import { MAX_CONCURRENT_QUERIES } from '../lib/utils.js'
+const log = debug('metrics:updateMetrics')
+
+/**
+ * @typedef {import('pg').Pool} Client
+ * @typedef {{ name: string, value: number }} Metric
+ */
+
+const COUNT_USERS = `
+SELECT COUNT(*) AS total
+FROM nftstorage.user u JOIN nftstorage.user_tag ut
+ON u.id = ut.user_id
+WHERE ut.tag = 'HasSuperHotAccess'::user_tag_type AND ut.value = 'true'
+`
```

Comment:
Isn't this just a count of `HasSuperHotAccess` tags?

```suggestion
const COUNT_USERS = `
SELECT COUNT(*) AS total
FROM nftstorage.user_tag
WHERE tag = 'HasSuperHotAccess'::user_tag_type AND value = 'true'
`
```